backend/env.json

```diff
@@ -5,7 +5,7 @@
 },
 "ALLOWED_ORIGIN" : {
   "type" : "object",
-  "default" : ["http://localhost:3000", "http://localhost:3000/", "http://localhost:3005", "http://localhost:3005/", "http://localhost:5050", "http://localhost:5050/"]
```

Comment:
We usually run the frontend at port 5050, can you add 5050 back into the default list?